packages/common/src/core/schemas/hubproject.ts

```diff
@@ -118,39 +129,58 @@ export const HubProjectEditUiSchema: IUiSchema = {
   type: "Layout",
   elements: [
     {
-      labelKey: "{{i18nScope}}.name.label",
-      scope: "/properties/name",
-      type: "Control",
-    },
-    {
-      labelKey: "{{i18nScope}}.summary.label",
-      scope: "/properties/summary",
-      type: "Control",
-      options: {
-        helperText: {
-          labelKey: "{{i18nScope}}.summary.helperText",
+      type: "Section",
+      options: { headerTag: "h4" },
+      labelKey: "{{i18nScope}}.basicInfo.label",
+      elements: [
+        {
+          labelKey: "{{i18nScope}}.name.label",
+          scope: "/properties/name",
+          type: "Control",
         },
-      },
-    },
-    {
-      labelKey: "{{i18nScope}}.description.label",
-      scope: "/properties/description",
-      type: "Control",
-      options: {
-        helperText: {
-          labelKey: "{{i18nScope}}.description.helperText",
+        {
+          labelKey: "{{i18nScope}}.summary.label",
+          scope: "/properties/summary",
+          type: "Control",
+          options: {
+            helperText: {
+              labelKey: "{{i18nScope}}.summary.helperText",
+            },
+          },
+        },
+        {
+          labelKey: "{{i18nScope}}.description.label",
+          scope: "/properties/description",
+          type: "Control",
+          options: {
+            helperText: {
+              labelKey: "{{i18nScope}}.description.helperText",
+            },
+          },
         },
-      },
+        {
+          labelKey: "{{i18nScope}}.status.label",
```

Comment:
So, looking back at my issue for status, I think it's actually supposed to be its own section. Sorry I didn't do that myself, but what I'd do here is create a new section with label "Status", and then not include a label on the actual field, i.e: 

```ts
{
  type: "Section",
  labelKey: "{{i18nScope}}.status.label",
  elements: [
    {
      scope: "/properties/status",
      type: "Control",
      options: {
        enum: {
           i18nScope: "{{i18nScope}}.status.enum"
        }
      }
    }
  ]
}
```

I also think this is supposed to go _after_ the location section